app-admin/etcd-wrapper/files/etcd-wrapper

```diff
@@ -20,11 +20,12 @@ function require_ev_one() {
     exit 1
 }
 
-require_ev_one ETCD_IMG ETCD_TAG
+require_ev_one ETCD_IMG ETCD_IMG_TAG
```

Comment:
This change is going to break people's clusters when they update. Is that okay? (no idea if anyone is actually using this)